pilot/pkg/bootstrap/server.go

```diff
@@ -591,33 +585,16 @@ func (s *Server) initDNSTLSListener(dns string, tlsOptions TLSOptions) error {
 
 // initialize secureGRPCServer - using DNS certs
 func (s *Server) initSecureGrpcServer(port string, keepalive *istiokeepalive.Options, tlsOptions TLSOptions) error {
-	certDir := dnsCertDir
-
-	key := model.GetOrDefault(tlsOptions.KeyFile, path.Join(certDir, constants.KeyFilename))
-	cert := model.GetOrDefault(tlsOptions.CertFile, path.Join(certDir, constants.CertChainFilename))
-
-	certP, err := tls.LoadX509KeyPair(cert, key)
+	// TODO(ramaraochavali): Watch root cert also?
```

Comment:
Is there a way to dynamically load CA?

Reply:
Not sure. That is why I left this TODO

Reply:
It is probably fine to just restart the server if the root cert is changed - should be a very rare event.